themes/devious-diamonds.omp.yaml

```diff
@@ -44,105 +43,123 @@ blocks:
         background: magenta
         foreground: black
         template: " {{ if .SSHSession }} {{ end }}{{ .UserName }}@{{ .HostName }} "
-      - type: angular
-        style: powerline
-        powerline_symbol: 
-        background: lightRed
-        foreground: black
-        properties:
-          fetch_version: true
-        template: " ﮰ {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} "
-      - type: aws
-        style: powerline
-        powerline_symbol: 
-        background: yellow
-        foreground: black
-        properties:
-          display_default: false
-        template: "  {{ .Profile }}{{ if .Region }}@{{ .Region }}{{ end }} "
-      - type: az
-        style: powerline
-        powerline_symbol: 
-        background: lightBlue
-        foreground: black
-        properties:
-          display_default: false
-        template: " ﴃ Subscription {{ .Name }} ({{ if .EnvironmentName | contains 'AzureCloud' }}{{ 'Global' }}{{ else }}{{ .EnvironmentName }}{{ end }}) "
-      - type: azfunc
-        style: powerline
-        powerline_symbol: 
-        background: yellow
-        foreground: black
-        properties:
-          display_mode: files
-          fetch_version: false
-        template: " ﴃ {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} "
-      - type: go
-        style: powerline
-        powerline_symbol: 
-        background: lightCyan
-        foreground: black
-        properties:
-          fetch_version: true
-        template: "  {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} "
-      - type: java
-        style: powerline
-        powerline_symbol: 
-        background: lightCyan
-        foreground: black
-        template: "  {{ .Full }}"
-      - type: julia
-        style: powerline
-        powerline_symbol: 
-        background: lightCyan
-        foreground: black
-        properties:
-          fetch_version: true
-        template: "  {{ if .Error }}{{ .Error }}{{ else }}{{ .Full }}{{ end }} "
-      - type: kubectl
-        style: powerline
-        powerline_symbol: 
-        background: lightYellow
-        foreground: black
-        template: " ﴱ {{.Context}}{{if .Namespace}} :: {{.Namespace}}{{end}} "
-      - type: node
-        style: powerline
-        powerline_symbol: 
-        background: lightGreen
-        foreground: black
-        properties:
-          fetch_version: true
-        template: "  {{ if .PackageManagerIcon }}{{ .PackageManagerIcon }} {{ end }}{{ .Full }} "
-      - type: php
-        style: powerline
-        powerline_symbol: 
-        background: lightCyan
-        foreground: black
-        template: "  {{ .Full }} "
+      # - type: angular
```

Comment:
this will annoy people who're using the theme. I wouldn't do this and toggle these locally instead of disabling them in the config, e.g. `oh-my-posh toggle angular`.

Reply:
Wow I'm embarrassed that I didn't consider that. I'll revert that today.

Reply:
@JanDeDobbeleer Done! Part of my motivation was the squashed rendering due to length [on the themes page](https://ohmyposh.dev/docs/themes#devious-diamonds) (and the invalid template text, which I hope I fixed). I figured if I commented out "unnecessary" blocks, it would trim that down to a more "usual" render. Oops!